### PR TITLE
Add protected workspace

### PR DESCRIPTION
**Pull request by GallaisPoutine**
Hi @pedroscaff

This PR is to merge the following option 'ignore'.
This is a feature asked in issue #20 and already implemented by @savchenko.
Can you please review this so it can be merged ?
Thanks,

Introduce 'ignore' option under [general] section.

This option allows to append a character at the end of the renamed workspace. All workspace with this ending character will not be renamed after.

### CODE REVIEW

**Comment by pedroscaff** [comment]
LGTM, can you please also add docs on how to use this option?

**Comment by GallaisPoutine** [comment]
@pedroscaff yes. Do you prefer that I send a v2, or just another patch later for the documentation ?

Review is still required tho. :)

**Comment by pedroscaff** [comment]
> @pedroscaff yes. Do you prefer that I send a v2, or just another patch later for the documentation ?
> 
> Review is still required tho. :)

please update this PR with the docs than I'll approve :slightly_smiling_face:

**Comment by GallaisPoutine** [comment]
@pedroscaff done :smile:

File: README.md [hunk 1]
```diff
@@ -114,6 +117,7 @@ You can take this a bit further by using a bar that trims the workspace number a
 
 ## Contributors
 * [Pedro Scaff (pedroscaff)](https://github.com/pedroscaff)
+* [Raphaël Gallais-Pou (GallaisPoutine)](https://github.com/GallaisPoutine)
```
Comment (pedroscaff) [inline README.md:120]: :heart:

**Review by pedroscaff — APPROVED** [review]

**Comment by GallaisPoutine** [comment]
Closes #20

**Pull request merged**